libwallet/assets_manager.go

```diff
@@ -960,3 +961,26 @@ func (mgr *AssetsManager) DeleteDEXData() error {
 	// Delete dex client db.
 	return os.Remove(dexDBFile)
 }
+
+func (mgr *AssetsManager) ListenAssetChange(listen func()) {
```

Comment:
it's not very clear what this does from the naming, also since it's an exported method, please add a comment to it.

is the primary purpose of this method to watch the asset balance? if so then how does this sound?

`WatchAssetBalance`